packages/onboard-react/src/components/HardwareWalletSteps/AddressListTable.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, ReactNode, useCallback, useMemo, useState } from "react";
+import {
+  Button,
+  AddressTablePagination,
+  TableBase,
+  ColumnOptions,
+  TransactionId,
+  Align,
+} from "@sovryn/ui";
+import { Account } from "@sovryn/onboard-hw-common";
+import { utils } from "ethers";
+import styles from "./AddressListTable.module.css";
+
+const PER_PAGE = 5;
+
+type AddressListTableProps = {
+  items: Account[];
+  onAccountSelected: (account: Account) => void;
+};
+
+type Item = {
+  index: string;
+  address: ReactNode;
+  balance: string;
+  asset: string;
+  account: Account;
+};
+
+export const AddressListTable: FC<AddressListTableProps> = ({
+  items,
+  onAccountSelected,
+}) => {
+  const [selected, setSelected] = useState<Account>();
+  const [offset, setOffset] = useState(0);
+
+  const handleSelect = useCallback(
+    (item: Item) => setSelected(item.account),
+    [setSelected]
+  );
+
+  const handleConfirm = useCallback(
+    () => onAccountSelected(selected!),
+    [selected, onAccountSelected]
+  );
+
+  const paginatedItems: Item[] = useMemo(
+    () =>
+      items.slice(offset, offset + PER_PAGE).map((item, index) => ({
+        index: String(offset + index).padStart(2, "0"),
```

Comment:
```suggestion
        index: String(offset + index + 1).padStart(2, "0"),
```
I added `+1` because we index from zero in the implementation but in the design, we index from 1 (and I think it's the correct way to do it).

Reply:
Derivation addresses must be indexed from 0, example in the design is wrong.

Reply:
That may be true but we have a custom `cellRenderer` on `index` column so it can show 1 even though the index is 0.